README.md

```diff
@@ -1,11 +1,41 @@
-# Autoscale AWS Library
+# FortiGate Autoscale on AWS
 
-This is an intial readme template.
+The implementation for FortiGate Autoscale using [Fortinet Autoscale](https://github.com/fortinet/autoscale-core) on Amazon AWS platfrom.
+
+This project provides a multi-group Auto Scaling functionality for Fortinet FortiGate EC2 instances to form an HA cluster with failover protection.
+
+This project has the following features:
+
+1. Multi-group Hybrid Licensing models:
+   1. **Hybrid**: 1 fix-sized Auto Scaling group of 2 (and more) BYOL FortiGate Instances, and 1 dynamically scalable Auto Scaling group of (0 or more) On-Demand FortiGate Instances.
+   2. **BYOL-Only**: 1 dynamically scalable Auto Scaling group of (0 or more) BYOL FortiGate Instances.
+   3. **PAYG-Only**: 1 dynamically scalable Auto Scaling group of (0 or more) On-Demand FortiGate Instances.
+2. AWS Transit Gateway Integration.
+
+## Deployment packages
+
+  * Download the **aws-cloudformation.zip** for the latest production version from [project release page](https://github.com/fortinet/fortigate-autoscale-aws/releases).
+  * or, manually generate the deployment package:
+    1. Chechkout the **main** branch of the project.
```

Comment:
```suggestion
    1. Check out the **main** branch of the project.
```

Reply:
approved.
    1. From the [project release page](https://github.com/fortinet/fortigate-autoscale-aws/releases), download the source code (.zip or .tar.gz) for the latest version.